deploy/crds/ocs.openshift.io_storageclusters_crd.yaml

```diff
@@ -1447,6 +1447,9 @@ spec:
                           phase:
                             description: Phase represents the current phase of PersistentVolumeClaim.
                             type: string
+                  devicetype:
+                    description: DeviceType is the value of device type in this StorageDeviceSet
```

Comment:
DeviceType represents the value of underlying storage device used (restricted to 3 values; SSD, NVME, HDD)